wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/SyntaxRules.kt

```diff
@@ -0,0 +1,26 @@
+package com.squareup.wire.schema
+
+import com.squareup.wire.schema.ProtoFile.Syntax
+import com.squareup.wire.schema.ProtoFile.Syntax.PROTO_2
+import com.squareup.wire.schema.ProtoFile.Syntax.PROTO_3
+
+/** A set of rules which defines schema requirements for a specific [Syntax]. */
+interface SyntaxRules {
+  fun enumRequiresZeroValueAtFirstPosition(): Boolean = false
```

Comment:
Would it be better to define this as `val` instead of `fun`? Thoughts?

Reply:
I don’t feel strongly either way. And you can always change it.